src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div>
+    <p>
+      Add the people and corporations/firms who will have a role in your company. People can have multiple roles;
+      Corporation/firms can only be Incorporators.
+    </p>
+    Your application must include the following:
+    <ul>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.COMPLETING_PARTY, 1, 'EXACT')" class="cp-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="cp-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>The Completing Party</span>
+      </li>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.INCORPORATOR, 1, 'ATLEAST')" class="incorp-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="incorp-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>At least one Incorporator</span>
+      </li>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.DIRECTOR, 1, 'ATLEAST')" class="dir-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="dir-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>At least one Director</span>
+      </li>
+    </ul>
+    <div class="btn-panel" v-if="orgPersonList.length === 0">
```

Comment:
This div needs a little more whitespace above it... maybe 1 rem padding?

Reply:
This button panel also needs more whitespace below it -- either bottom margin for this div or top margin for the divs below it.

Also, I think the Add Person / Add Corporation containers (v-cards) have too much padding. I think they should look like the list container. Try padding: 0.

Reply:
Fixed!